draft-ietf-quic-transport.md

```diff
@@ -726,16 +727,17 @@ Connection ID.
 
 ### Tokens
 
-If the client has an unused token that it received in a NEW_TOKEN frame on a
-previous connection to what it believes to be the same server, it includes that
-value in the Token field of its Initial packet.
+If the client has an token received in a NEW_TOKEN frame on a previous
+connection to what it believes to be the same server, it can include that value
+in the Token field of its Initial packet.
 
-A client MUST NOT reuse a token if it believes that its point of network
-attachment has changed; that is, if there is a change in its local IP address or
-network interface.  Reusing a token on different network paths would allow
-activity to be linked between paths (see {{migration-linkability}}).  A client
-needs to start the connection process over if it migrates prior to completing
-the handshake.
+A client SHOULD NOT reuse a token; reused tokens enable activity on connections
+to be correlated.  A client MUST NOT reuse a token if it believes that its point
+of network attachment has changed; that is, if there is a change in its local IP
+address or network interface.  Reusing a token on different network paths would
+allow activity to be linked between paths (see {{migration-linkability}}).  A
+client needs to start the connection process over if it migrates prior to
+completing the handshake.
 
```

Comment:
Two separate sentences about linking, one stating "correlate", the other "linked".

Reply:
Oops, thanks.